e2e/support/helpers/e2e-ad-hoc-question-helpers.js

```diff
@@ -110,18 +110,35 @@ export function openTable({
   );
 }
 
+/**
```

Comment:
This empty area is not present in designs.

![image](https://github.com/metabase/metabase/assets/6830683/6a3b0657-2e8b-4d98-bfa6-06505a3de052)

Reply:
True. But that is not the concern of this milestone.

Reply:
https://github.com/metabase/metabase/issues/40668